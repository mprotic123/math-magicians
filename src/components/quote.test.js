import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './quote.js';

it('Renders Quote', () => {
  const quote = renderer.create(
    <Quote />,
  ).toJSON();
  expect(quote).toMatchSnapshot();
});