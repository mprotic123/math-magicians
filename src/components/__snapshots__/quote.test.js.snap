// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Quote 1`] = `
<div
  className="quote"
>
  <h1>
    The Quote
  </h1>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
    <br />
     -Lorem Ipsum
  </p>
</div>
`;
